creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -238,15 +238,68 @@ spec:
     pod: {}
 ----
 
+______________________________________________________________________________
+**Note:** If a specific MAC address is configured in the virtual machine
+interface spec, it's passed to underlying CNI plugin that should configure
+backend to allow for this particular address. While some plugins are known to
+work without special support for custom MAC addresses, others do (for example,
```

Comment:
"others do**n't**"?

The OpenShiftSDN example is maybe too specific. Do you think it would be worth saying that primary network plugins (calico, flannel, openshiftSDN) does not support it? However, then we should double check that they indeed don't.

Reply:
I will fix the typo, thanks.

I haven't checked whether `calico` or `flannel` support `mac` attribute. I suspect they don't do it natively but since they are netlink based they should support it through `tuning` plugin.

Reply:
The problem would not be that `tuning` does not change the MAC, but that it would break the connection (similar to OpenShiftSDN). I think it would be valuable for users to get simple **supported/not supported** list. If you think it is not a lot of work.

Reply:
@phoracek do we even test with `calico`? How do we define the list of plugins we would like to list here? There are tens of different plugins. Should we document each of them?

For `flannel`, I guess it makes sense to have it documented because it's quite popular.

Reply:
You are right. Let's do it just for Flannel which is tested under kubevirt.

Reply:
I've checked that `flannel` works with `tuning` plugin. I've changed wording here somewhat; there is an explicit list of plugins, classified by their MAC support status, below this section. I hope it's better now.